.ci.yaml

```diff
@@ -1,137 +1,8 @@
-# Describes the targets run in continuous integration environment.
-#
-# Flutter infra uses this file to generate a checklist of tasks to be performed
-# for every commit.
-#
-# The recipes mentioned below refer to those in this repo:
-#   https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/
-#
-# The "flutter_drone" recipe just defers to dev/bots/test.dart in this repo,
-# with the shard set according to the "shard" key in this file.
-#
-# More information at:
-#  * https://github.com/flutter/cocoon/blob/main/CI_YAML.md
+# This is intentionally a subset of .ci.yaml from the master branch.
 enabled_branches:
-  - master
-  - flutter-\d+\.\d+-candidate\.\d+
+  - ios-experimental
 
 platform_properties:
-  staging_build_linux:
-    properties:
-      dependencies: >-
-        [
-          {"dependency": "curl", "version": "version:7.64.0"}
-        ]
-      os: Ubuntu
-      cores: "8"
-      device_type: none
-      ignore_flakiness: "true"
-  linux:
```

Comment:
I think the `linux` platform_properties needs to be kept for `Linux analyze` to work

Reply:
Actually I'll get rid of this too (unless you really want it) - it means we need to build a linux engine which seems unnecessary at this point.